Microsoft.Toolkit.Uwp.UI.Media/Geometry/Core/RegexFactory.cs

```diff
@@ -0,0 +1,607 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+
+[assembly: InternalsVisibleTo("UnitTests.UWP")]
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Geometry.Core
+{
+    /// <summary>
+    /// Contains all the Regular Expressions which are
+    /// used for parsing the CanvasGeometry Mini-Language
+    /// </summary>
+    internal static class RegexFactory
+    {
+        // Whitespace
+        private const string Spacer = @"\s*";
+
+        // Whitespace or comma
+        private const string SoC = @"(?:\s+|\s*,\s*)";
```

Comment:
Can we use fuller more descriptive names here so it's clearer when they're being used what they're looking for?

Reply:
Renamed it to `SpaceOrComma`